UserInterface/GameManager/GameManager.cs

```diff
@@ -69,43 +90,85 @@ private void RunGame()
                 Console.CursorVisible = false;
                 Pair? currentPair = _pairManager.Pair;
 
-                _board.FillBoardWithAnimals(_animals);
+                _board.FillBoardWithObjects(_gameObjects);
 
                 _userInterface.DrawBoard(_board);
-                _board.RemoveAnimalFromBoard(_animals);
-
-                _animalActions.Move(_board, _animals);
+                _board.RemoveGameObjectFromBoard(_gameObjects);
```

Comment:
You just added objects to board. Why you are removing them? 
Method probably does something else not just removes objects. In such case rename method so that it is clear what it does.

Reply:
Done.